imageRegistry.py

```diff
@@ -274,9 +291,14 @@ def podman_authenticate(self) -> None:
         """
         Login to the in-cluster registry using Podman on the host with a token.
         """
+        SECONDS = 1
+        MINUTES = SECONDS * 60
```

Comment:
use timer.py in this repo. Then you can pass in a string like "6m". Please do that in a follow up since we need this PR in asap.